generators/chipyard/src/main/scala/TestSuites.scala

```diff
@@ -64,91 +63,11 @@ class TestSuiteHelper
   def addSuites(s: Seq[RocketTestSuite]) { s.foreach(addSuite) }
 
   /**
-  * Add BOOM tests (asm, bmark, regression)
+  * Add generic tests (asm, bmark, regression) for all cores.
```

Comment:
This comment is bit misleading if we make this function generic.
We only add tests for the core with hartid=0 (since only hart0 runs the asm tests).

Reply:
Agreed. We never really talk about this anywhere in the documentation either. I think it would be nice to add a better comment to this file and maybe add the "only runs on 1st hart" to the documentation.

Reply:
Sure. I will fix this comment in the next commit.